### DISCUSSION TITLE
Contribution is not count in my profile!

### DISCUSSION THREAD

**Discussion by NextGenGk** [community/community#59425] [Other Feature Feedback, Questions, & Ideas] [answered]
My contribution is not shown in my profile? 
When I am pushing a changes in our github repo, contribution is not count in my profile 
Please help me

**Comment by Saliya99** [comment, marked as answer]
If your contributions are not showing up on your GitHub profile after pushing changes to a repository, here are a few things you can check:
Verify the email address associated with your GitHub account: GitHub uses the email address linked to your commits to identify your contributions. Ensure that the email address you're using to make commits matches the one associated with your GitHub account. You can check your email settings in your GitHub account settings.

Confirm that your commits are linked to your GitHub account: When you make a commit, it should be associated with your GitHub account. Check the authorship of your commits by running the following command in the repository's directory:
----------------
git log
----------------
Make sure that the author name and email match the ones linked to your GitHub account.

Check the repository's visibility and access: Contributions will only appear on your GitHub profile if you have the necessary permissions to view and contribute to the repository. Ensure that you have the appropriate access level (e.g., write access) for the repository where you are pushing your changes.

Wait for the contribution activity to update: Sometimes, it may take a short period for the contribution activity to update on your GitHub profile. GitHub's contribution graphs are typically refreshed periodically, so you might need to wait a little while for your contributions to appear.

Check the repository settings: In some cases, the repository settings might be configured to exclude certain types of contributions from being counted towards your profile. For instance, if the repository is set to "private" or if you have disabled the "Include private contributions" option in your profile settings, your contributions to private repositories may not be displayed.

If none of these steps resolve the issue, you can reach out to GitHub support for further assistance. They can help investigate the specific issue you're facing and provide guidance to ensure your contributions are properly reflected on your GitHub profile.

**Reply by NextGenGk** [reply]
Thnk u for helping me!! @Saliya99

**Comment by Divyanshu9822** [comment]
I'm facing this issue for a while. My whole graph is based on contributions to private repo. I've verified my email address from whcich i'm commiting and the same is used in github acc also, other than this i found that i've raised an issue in same repo and that came up in graph but not the commits i did in repo although few previous commits are already there in PR and is counted but after that no contribution is showing up in any repo (private) i'm contributing too

**Reply by Rini-Tikki** [reply]
I am facing exactly issue, did you find any resolution to this @Divyanshu9822?
